infrastructures/docker/src/main/java/org/eclipse/che/workspace/infrastructure/docker/output/OutputService.java

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2017 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.workspace.infrastructure.docker.output;
+
+import org.eclipse.che.api.core.jsonrpc.commons.RequestHandlerConfigurator;
+import org.eclipse.che.api.core.notification.RemoteEventService;
+import org.eclipse.che.api.workspace.shared.dto.event.InstallerLogEvent;
+import org.eclipse.che.api.workspace.shared.dto.event.MachineLogEvent;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+/**
+ * Service for handling output.
```

Comment:
Since service and handling are pretty common word javadocs are not very clear to me. Can you describe more precisely the purpose of the class in javadocs?